src/domain/balances/entities/balance.entity.ts

```diff
@@ -23,6 +23,11 @@ export const Erc20BalanceSchema = z.object({
 
 const FiatSchema = z.object({
   fiatBalance: z.string().nullish().default(null),
+  fiatBalance24hChange: z.coerce
+    .string()
+    .nullable()
+    // Transaction Service does not return this field
```

Comment:
Nit: the Transaction Service doesn't return any of the fields in FiatSchema. We can move this comment to FiatSchema itself, but I'm not sure it's relevant here (ideally, this class is agnostic regarding the origin of the balances data).

Reply:
True - this was a remnant from development. Removed in a2226a9401f4bb00fc7cf36cae976c022790e3d6.